egs/thchs30/s5/local/dae/run_dae.sh

```diff
@@ -0,0 +1,154 @@
+#!/bin/bash
+
+#dnn dae training
+
+dwntest=false
+stdtest=false
+stage=0
+nj=8
+
+. ./cmd.sh ## You'll want to change cmd.sh to something that will work on your system.
+           ## This relates to the queue.
+
+. ./path.sh ## Source the tools/utils (import the queue.pl)
+. utils/parse_options.sh || exit 1;
+
+
+#generate noisy data. We focuse on the 0db condition.
+#For training set, generate noisy data with SNR mean=0, variance=10, with three noise types mixed together.  
+#For dev, generate noisy data with SNR mean=0, variance=0, with three niose types mixed together
+#For test, either use the standard test set (stdtest=true) or randomly generated data (stdtest=false)
+#The standard noisy test data are availabe at the challenge website (by setting dwntest=true):
+#http://data.cslt.org/thchs30/challenges/asr.html
+
+if [ $stage =  0 ]; then
+   # generate training data
+   noise_scp=local/dae/noise.scp
+   noise_prior="0.0,10.0,10.0,10.0" #define noise type to sample. [S_clean, S_white, S_car, S_cafe]
+   noise_level=0 #0db condition
+   sigma0=10 #some random in SNR
+   seed=32
+   verbose=0
+
+   addnoise_opt="--noise-level $noise_level --sigma0 $sigma0 --seed $seed --verbose $verbose --noise-prior $noise_prior --noise-src $noise_scp --wavdir NULL"
+
+   rm -rf data/dae/train && mkdir -p data/dae/train || exit 1
+   cp -L data/fbank/train/{wav.scp,spk2utt,utt2spk,text} data/dae/train || exit 1
+   local/dae/make_fbank.sh --nj $nj --cmd "$train_cmd"  \
+   --addnoise-opt """$addnoise_opt"""  \
```

Comment:
you should indent lines that appear after backslash to make clear that they are part of the previous command.